grails-app/services/org/pih/warehouse/inventory/CycleCountService.groovy

```diff
@@ -99,4 +101,30 @@ class CycleCountService {
             criteria.createAlias("product", "product", JoinType.INNER_JOIN)
         }
     }
+
+    List<CycleCountRequest> createRequest(CycleCountRequestCommand command) {
```

Comment:
I'd do `createRequests` since this is creating multiple

Reply:
yeah, forgot to change it, I had the same feeling at some point, thanks for pointing that out